README.rst

```diff
@@ -561,7 +586,7 @@ to file a new bug or to add yourself to the CC list for an existing bug
 You can explore the list of existing bugs, which may include workarounds
 for the problem you've run into on GitHub Issues:
 
-        https://github.com/SCons/scons/issues
+        httpss://github.com/SCons/scons/issues
```

Comment:
typo httpss

Reply:
yeah, must have changed one too many, getting the "http" from one that was already "https".